### DISCUSSION TITLE
Add Master for Tidal

### DISCUSSION THREAD

**Discussion by jonathanmcohen** [RandomNinjaAtk/docker-lidarr-extended#31] [Ideas]
Is it possible to add `tidal-dl -q Master` as an option for Tidal-DL

**Comment by RandomNinjaAtk** [comment]
From my previous testing, Master doesn't work, so there is no point.

**Comment by jonathanmcohen** [comment]
Thanks for the quick response.

**Comment by enigmaoftech** [comment]
It looks like the master download has a bug in it. It used to work. https://github.com/yaronzz/Tidal-Media-Downloader/issues/990
When it is fixed could this be something that is added?

By the way great all in one tool! Thank you!

**Reply by RandomNinjaAtk** [reply]
If I had to make a prediction, probably not. I believe authenticating is a big issue for that format.

**Reply by enigmaoftech** [reply]
Sounds good, I can always download the ones I want manually and add. On the auth part I'm going to play around with this and see if I can't get it to auto authorize itself. This would save having to go back in every week.

**Comment by enigmaoftech** [comment]
In your script what API version are you using? If switched to use Android Auto number 4 it will download master track audio.
In the get

[INFO] Current Version: 1.0.1 GIST
-------------API-KEYS---------------
+-------+-------+--------------+-----------------------------+
| Index | Valid | Platform     | Formats                     |
+-------+-------+--------------+-----------------------------+
| 0     | False | Fire TV      | Normal/High/HiFi(No Master) |
| 1     | True  | Fire TV      | Master-Only(Else Error)     |
| 2     | False | Android TV   | Normal/High/HiFi(No Master) |
| 3     | False | TV           | Normal/High/HiFi/Master     |
| 4     | True  | Android Auto | Normal/High/HiFi/Master     |
+-------+-------+--------------+-----------------------------+

**Reply by RandomNinjaAtk** [reply]
It already uses 4, see this thread: https://github.com/RandomNinjaAtk/docker-lidarr-extended/issues/134
